accesscontrol/jwt.go

```diff
@@ -139,7 +147,12 @@ func (j *JWT) Validate(req *http.Request) error {
 		return errors.JwtTokenMissing.Message("token required")
 	}
 
-	token, err := j.parser.ParseWithClaims(tokenValue, jwt.MapClaims{}, j.getValidationKey)
+	parser, err := newParser(j.algorithm, claims)
+	if err != nil {
+		return err
+	}
+
+	token, err := parser.ParseWithClaims(tokenValue, jwt.MapClaims{}, j.getValidationKey)
```

Comment:
```suggestion
	token, err := parser.Parse(tokenValue, j.getValidationKey)
```
In general I would prefer not creating the parser every time again. There are options to pass current Claims to an existing one and even the Claims are interfaced to call Valid() with our options. However, in favour for using another package in the future its ok for now.